content/english/useless-box/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Useless box: Silly gadget circuit"
+description: "A guide to build a 'useless box' to learn the basics of circuits and how a battery powers action."
+date: 2020-09-207T11:45:38-07:00
+prereq: "None"
+difficulty: "Beginner"
+download: ""
+draft: true
+icon: "fas fa-microchip"
```

Comment:
Can you find a font awesome icon that best suits this workshop?

https://fontawesome.com/search

Reply:
sounds good, I think I'll go with https://fontawesome.com/icons/microchip?s=solid&f=classic